documentation/troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Troubleshooting Kubernetes vSphere Cloud Provider
+---
+This document is aimed to help debug issues in the Kuberentes vSphere Cloud Provider.
+
+For general Kubernetes Issue, Please visit [https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/)
```

Comment:
For general Kubernetes Issue, Please -> To troubleshoot issues with Kuberenetes cluster deployment, please

Reply:
fixed.